allure-generator/src/main/javascript/components/marks-toggle/styles.scss

```diff
@@ -0,0 +1,32 @@
+.marks-toggle {
+  &__items {
+    display: block;
+    user-select: none;
+    word-break: break-word;
+  }
+  &__item {
+    cursor: pointer;
+    padding: 5px 0 5px 0;
+    display: inline-block;
+    white-space: nowrap;
+  }
+}
+
+.y-label-mark {
```

Comment:
use modifier instead of different classes

Reply:
Did likewise like `status-toggle`. How to do it right, can you give an example?